test/elements/depayloader_test.exs

```diff
@@ -18,23 +17,20 @@ defmodule DepayloaderTest do
       "[frameid:1e][timestamp:1] you?"
     ]
 
-    options = %Pipeline.Options{
-      elements: [
-        source: %Source{output: inputs, caps: %Packet{type: :custom_packets}},
-        depayloader: %Depayloader{packets_per_frame: 5},
-        sink: Sink
+    structure = [
+        child(:source, %Source{output: inputs, stream_format: %Packet{type: :custom_packets}}),
+        child(:depayloader, %Depayloader{packets_per_frame: 5}),
+        child(:sink, Sink),
+        get_child(:source) |> get_child(:depayloader) |> get_child(:sink)
       ]
-    }
 
-    {:ok, pipeline} = Pipeline.start_link(options)
-    Pipeline.play(pipeline)
+    pipeline = Pipeline.start_link_supervised!(structure: structure)
     assert_start_of_stream(pipeline, :sink)
 
     assert_sink_buffer(pipeline, :sink, %Buffer{payload: "Hello! How are you?"})
 
     assert_end_of_stream(pipeline, :sink)
     refute_sink_buffer(pipeline, :sink, _, 0)
-    Pipeline.stop_and_terminate(pipeline, blocking?: true)
```

Comment:
We can stop the pipeline with: https://hexdocs.pm/membrane_core/0.12.5/Membrane.Testing.Pipeline.html#terminate/2

Reply:
This one also applies to some other similar cases in other tests